frontend/src/Onboarding/OnboardingInfoScreen.tsx

```diff
@@ -337,21 +330,21 @@ const mapDispatchToProps = (dispatch: Dispatch) => ({
     dispatch(setAcademicYearAction(academicYear)),
   setGraduationYear: (academicYear: number) =>
     dispatch(setGraduationYearAction(academicYear)),
-  setMajor: (major?: Major) => dispatch(setDeclaredMajorAction(major)),
-  setCoopCycle: (coopCycle: string, plan: Schedule) =>
-    dispatch(setCoopCycle(coopCycle, plan)),
+  setMajor: (major?: Major) => dispatch(setUserMajorAction(major?.name || '')),
```

Comment:
We only store the major name in redux (to stay consistent with the backend), and we can always use `findMajorFromName` if we need the object